js/controllers/search.js

```diff
@@ -44,15 +44,13 @@ SearchController.prototype.onSearchButton_ = function() {
   return false;
 };
 
-SearchController.prototype.onChangeFocus_ = function() {
-  if (document.activeElement === document.body ||
-      $(document.activeElement).parents('.search-container').length) {
-    return;
+SearchController.prototype.onFocusoutSearch_ = function(e) {
```

Comment:
I would suggest, if possible, a name that says what this function does, rather than where it's going to be called from.